index.html

```diff
@@ -4,9 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <div class="main-container">
+    <header>
+        <nav class="navbar">
+            <h2 class="logo">Anelisa Didishe</h2>
+            <div id="menu">
+                <i class="fa fa-bars"></i>
+            </div>
+        </nav>
+    </header>
+    <div class="container">
+        <h1>Hey There. I’m Anelisa</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      
+        <div class="social-icon" id="social-icon">
+          <ul>
+            <li class="social">
+              <a href="#"><i class="fa fa-github"></i></a>
+            </li>
+            <li class="social">
+              <a href="#"><i class="fa fa-linkedin"></i></a>
+            </li>
+            <li class="social">
+                <a href="#"><i class="fa fa-hand-peace-o"></i></a>
+            </li>
+            <li class="social">
+              <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li class="social">
+              <a href="#"><i class="fa fa-medium"></i></a>
+            </li>
+          </ul>
+        </div>
+    </div>

```

Comment:
- [ ] The container of the section is a `section` tag. It represents a generic standalone section of the document, please kindly use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info 😁 😁